TestCases/turbomachinery/Aachen_turbine/aachen_3D_MP_restart.cfg

```diff
@@ -0,0 +1,371 @@
+LINEAR_SOLVER_PREC= LU_SGS
+%
+% Min error of the linear solver for the implicit formulation
+LINEAR_SOLVER_ERROR= 1E-4
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 15
+%
+% ----------------------- SLOPE LIMITER DEFINITION ----------------------------%
+%
+% Coefficient for the limiter
+VENKAT_LIMITER_COEFF= 0.01
+%
+% Freeze the value of the limiter after a number of iterations
+LIMITER_ITER= 999999
+%
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+% Convective numerical method (JST, LAX-FRIEDRICH, CUSP, ROE, AUSM, HLLC,
+%                              TURKEL_PREC, MSW)
+CONV_NUM_METHOD_FLOW= ROE
+ENTROPY_FIX_COEFF= 0.001
+%
+JST_SENSOR_COEFF= ( 0.5, 0.25 )
+% Spatial numerical order integration (1ST_ORDER, 2ND_ORDER, 2ND_ORDER_LIMITER)
+MUSCL_FLOW= NO
+%
+% Slope limiter (VENKATAKRISHNAN, VAN_ALBADA)
+SLOPE_LIMITER_FLOW= VENKATAKRISHNAN
+%
+%
+% Time discretization (RUNGE-KUTTA_EXPLICIT, EULER_IMPLICIT, EULER_EXPLICIT)
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+%
+% -------------------- TURBULENT NUMERICAL METHOD DEFINITION ------------------%
+%
+% Convective numerical method (SCALAR_UPWIND)
+CONV_NUM_METHOD_TURB= SCALAR_UPWIND
+%
+% Spatial numerical order integration (1ST_ORDER, 2ND_ORDER, 2ND_ORDER_LIMITER)
+MUSCL_TURB= NO
+%
+% Slope limiter (VENKATAKRISHNAN, MINMOD)
+SLOPE_LIMITER_TURB= VENKATAKRISHNAN
+%
+% Time discretization (EULER_IMPLICIT)
+TIME_DISCRE_TURB= EULER_IMPLICIT
+%
+% Reduction factor of the CFL coefficient in the turbulence problem
+CFL_REDUCTION_TURB= 0.1
+%
+% ----------------------- DESIGN VARIABLE PARAMETERS --------------------------%
+%
+% Kind of deformation (NO_DEFORMATION, TRANSLATION, ROTATION, SCALE,
+%                      FFD_SETTING, FFD_NACELLE
+%                      FFD_CONTROL_POINT, FFD_CAMBER, FFD_THICKNESS, FFD_TWIST
+%                      FFD_CONTROL_POINT_2D, FFD_CAMBER_2D, FFD_THICKNESS_2D, FFD_TWIST_2D,
+%                      HICKS_HENNE, SURFACE_BUMP)
+DV_KIND= NO_DEFORMATION
+% 
+% Marker of the surface in which we are going apply the shape deformation
+DV_MARKER= (BLADE1, BLADE2, BLADE3)
+%
+% Parameters of the shape deformation
+DV_PARAM= ( 1, 0.5)
+%
+% Value of the shape deformation
+DV_VALUE= 0.01
+%
+% --------------------------- CONVERGENCE PARAMETERS --------------------------%
+%
+% Number of total iterations
+OUTER_ITER=10
+%
+% Convergence criteria (CAUCHY, RESIDUAL)
+CONV_FIELD=RMS_ENERGY[0] 
+%
+% Min value of the residual (log10 of the residual)
+CONV_RESIDUAL_MINVAL= -12
+%
+% Start convergence criteria at iteration number
+CONV_STARTITER= 10
+%
+% Number of elements to apply the criteria
+CONV_CAUCHY_ELEMS= 999
+%
+% Epsilon to control the series convergence
+CONV_CAUCHY_EPS= 1E-6
+%
+% Function to apply the criteria (LIFT, DRAG, NEARFIELD_PRESS, SENS_GEOMETRY, 
+% 	      	    		 SENS_MACH, DELTA_LIFT, DELTA_DRAG)
+%CAUCHY_FUNC_FLOW= DRAG
```

Comment:
You can remove the Cauchy convergence options in both files as they are unused

Reply:
Just done it